### DISCUSSION TITLE
Feature request: jump method

### DISCUSSION THREAD

**Discussion by yongtaoer** [hrynko/vue-pdf-embed#139] [Q&A] [answered]
# For example, as the browser browsed pdf components by default, you could go to a specific page and scroll to other pages, rather than having to change the page and not scroll to other pages of pdf.
<img width="1197" alt="image" src="https://user-images.githubusercontent.com/35984856/235043472-3378faea-6bd8-4d62-83da-55c499370db3.png">

**Comment by yongtaoer** [comment]
Also, as I scroll, how do I know which page belongs to the total number of pages

**Comment by hrynko** [comment]
Hi @YTaoger,

Sorry, I don't really understand your message. Could you rephrase?

**Comment by yongtaoer** [comment]
> Hi @YTaoger,
> 
> Sorry, I don't really understand your message. Could you rephrase?

I'm sorry, maybe it's my expression.
Currently, if page is not specified, all pages are viewed. But can you use parameters or methods to jump to a specific page while browsing? And, when scrolling, you can get which page is currently visible in the window?

**Comment by yongtaoer** [comment]
https://user-images.githubusercontent.com/35984856/235081877-5486260e-57df-4207-a2c5-506ed54c24ef.mp4

@hrynko Excuse me, in the video, notice the change of the top page, according to the page scroll, the current page number will also change, how to achieve this effect in `vue-pdf-embed`? You can also switch the current page number.

**Reply by hrynko** [reply, marked as answer]
You can set an ID to the `vue-pdf-embed` component, so that it is assigned to the container element and each page has an ID as `<PROVIDED_ID>-<PAGE_NUMBER>`. You can then leverage the anchor links for navigation and the [intersection observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) to display the current page number.

You can find some more insight in [this article](https://css-tricks.com/sticky-table-of-contents-with-scrolling-active-states/).

**Reply by yongtaoer** [reply]
> You can set an ID to the `vue-pdf-embed` component, so that it is assigned to the container element and each page has an ID as `<PROVIDED_ID>-<PAGE_NUMBER>`. You can then leverage the anchor links for navigation and the [intersection observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) to display the current page number.
> 
> You can find some more insight in [this article](https://css-tricks.com/sticky-table-of-contents-with-scrolling-active-states/).

Thank you for your answer ！

**Comment by yongtaoer** [comment]
@hrynko Hi, I wonder, why not use disableAutoFetch from pdf.js. I'm having a problem loading PDF fragments. How can I optimize it？

**Reply by hrynko** [reply]
I think it can be used as it is part of the init params. What problems are you experiencing?

**Reply by yongtaoer** [reply]
I am having problems loading PDF fragments. The current problem is loading the whole PDF when initializing it, but I want to be able to control whether the PDF is lazy loading or not.
